devimint/src/external.rs

```diff
@@ -845,7 +845,8 @@ pub async fn open_channel(
         .bech32
         .context("bech32 should be present")?;
 
-    bitcoind.send_to(cln_addr, 100_000_000).await?;
+    let txid = bitcoind.send_to(cln_addr, 100_000_000).await?;
+    bitcoind.poll_get_transaction(txid).await?;
```

Comment:
If this ever fails, we should just move polling for transaction into `send_to` itself?

Reply:
I think that would be a good idea.